### ISSUE TITLE
Email reminders of events

### ISSUE THREAD

**Issue by mark-adigital** [verbb/Events#95]
**What are you trying to do?**
We've been asked whether it would be possible to automate email reminders for those who have purchased a ticket the day before an event.

**What's your proposed solution?**
I'm aware this would be a significant bolt on to Events, and may be better handled by the email options already in Commerce - so maybe a way of batch updating the status of orders linked to purchased tickets at the `/events/purchased-tickets` screen would be the solution here? Then the email template could be written in Twig and configured through Commerce as linked to that status. There would still be a manual step in updating the status, but that's still a lot easier than the process now - or if there was a way to automate the status change to a selected status at a predefined interval before the event, then that would be even better.

**Comment by engram-design** [comment]
Yeah, emails are starting to get into the realm of Commerce, where they're managed. In similar situations, we've added the creation of reminder emails completely custom through a module (circumventing Commerce's emails).

The issue with order statuses is that not every purchased ticket is related to an order (you can create them manually or programatically, separate to an order). It's also probably no solving it in a nice way (having to create the statuses, admins need to go change it to send out emails).

The reality will probably be an entire "reminder" function, based off conditions (X days/months/etc from ticket dateCreated/ticket datePurchased/event startDate/etc). Will have to mull it over!

**Comment by mark-adigital** [comment]
Yeah - I think to get the maximum benefit you would need to be able to automate the status update to a selected status a set amount of time before the event really to remove the manual element. As you say though, if not all tickets are linked to an order then even that wouldn't catch all tickets.

Yes, you would need to offer more options than just 'One day before event' - that might work for our client but certainly not everyone! Maybe the email that is being sent could still be configured within Commerce's System Settings and use Commerce methods to actually send out? Then you're only interested in allowing the selection of that email and when/who the email should be sent to, rather than reinventing the wheel of configuring the email and template itself.
